### PR TITLE
Popup Modal

### PR DESCRIPTION
**Pull request by danutibe07**
popup modal for works section

### CODE REVIEW

**Review by Olamarx — CHANGES_REQUESTED** [review]
Hi @danutibe07,

Good job so far, indeed you have spent time on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- Linters passed ✔️
- Popup ✔️

## Required Changes ♻️
_Check the comments under the review_.
### Note that I would have made the comments inline but I see you merged most of your work already.
- [ ] I would recommend you add javascript linters to it, I see you have html and css linters already. You can find it [here](https://github.com/microverseinc/linters-config/tree/master/javascript). With this, you will be guided on best practices of Js usage. 
- [ ] I would recommend you display the whole work section dynamically with javascript because this is the requirement and it would help you to understand how to render data dynamically with js. You can have the whole data in an array and map over it with the map keyword.
- [ ] It is also required you use Js to display the whole popup content and you can use any of the HTML rendering keywords provided by js such as innerHTML, insertAdjacentHTML, or any other. This would help you know how to take full control of DOM manipulation with JS.
- [ ] I would recommend you stick to the style guide of the Figma design esp. the mobile responsiveness. At the moment, your popup is not following the font, margin, and padding styles.
### Your Design
image
### Figma Design
image
- [ ] Whenever I click on the close modal (X), it makes another popup at the footer of the site, I would recommend you check the close button functionality again. Attached is the image of another popup it makes.
image

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 1]
```diff
@@ -650,6 +620,136 @@ article h1 {
...
+    width: 100%;
+    padding-right: 40px;
+  }
+
+  .modal .close {
+    font-size: 50px;
+    width: 50px;
+    height: 50px;
+    color: #172b4d;
+    background-color: #e5e5e5;
```
Comment (franclobo) [inline style.css:669]: - [ ] Kindly correct the background color from the close button as the last code reviewer suggested 👇🏽 

> I would recommend you stick to the style guide of the Figma design esp. the mobile responsiveness. At the moment, your popup is not following the font, margin, and padding styles.

File: style.css [hunk 2]
```diff
@@ -650,6 +620,136 @@ article h1 {
...
+  .modal {
+    background-color: #e5e5e5;
+    height: 100%;
+    width: 100%;
+    position: fixed;
+    top: 0;
+    left: 0;
+    justify-content: center;
+    overflow: scroll;
+  }
```
Comment (franclobo) [inline style.css:635]: - [ ] Kindly correct all the modal styles as requested by the last code reviewer 👇🏽 

> I would recommend you stick to the style guide of the Figma design esp. the mobile responsiveness. At the moment, your popup is not following the font, margin, and padding styles. 

| Figma design | Your design |
|---|---|
| image | image |

**Review by franclobo — CHANGES_REQUESTED** [review]
Hi @danutibe07, 

# ♻️ Required changes ♻️ 

so-close-jim-halpert

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights

- Correct Github flow ✔️ 
- Desktop modal looks good at 1440px ✔️ 


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 3]
```diff
@@ -650,6 +622,136 @@ article h1 {
...
```
Comment (TadesseDev) [inline style.css:635]: - [ ] Dear @danutibe07, Good job adding Most of the changes, But as you can see and as requested by the previous reviewer your modal needs more alignment with the given Figma template. Specially for the margin and the padding (Spacing between elements).

### Figma design 

### Your design 
Screenshot 2023-02-10 083025

File: style.css [hunk 4]
```diff
@@ -650,6 +622,136 @@ article h1 {
...
+    width: 100%;
+    padding-right: 40px;
+  }
+
+  .modal .close {
+    font-size: 50px;
+    width: 50px;
+    height: 50px;
+    color: #172b4d;
+    background-color: #e5e5e5;
```
Comment (TadesseDev) [inline style.css]: - [ ] Dear @danutibe07, As the previous code reviewer mentioned,  Kindly correct the background color from the close button. 👇🏽
### Figma design 
217996592-93dad669-d3ac-40e5-92b0-67a39a4ce412

### Your design 
Screenshot 2023-02-10 083025

**Review by TadesseDev — CHANGES_REQUESTED** [review]
### Hi @danutibe07 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

199733072-bdacca19-8df9-43a7-be93-3aff2f1c15c8

 ## Highlights
 - [ ] Good job making most of the changes requested by the previous reviewers. 🙏
 - [ ] Good job overall following the template design 💯
 - [ ]  Correct GitHub Flow 👍🏾
 - [ ]  All linters passed 🥇

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by pinkmoon25 — APPROVED** [review]
Hi @danutibe07 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
Linters :heavy_check_mark: 
Popup :heavy_check_mark: 
Github flow :heavy_check_mark:  

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**